### PR TITLE
v13.2.1

### PR DESCRIPTION
**Pull request by pouladpld**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
 test: off
 deploy:
   - provider: NuGet
-    server: https://www.myget.org/F/telegram-bot/api/v2/package
+    server: https://www.myget.org/F/telegram-bot-core/api/v2/package
     api_key:
-      secure: Dl2/cPfGdESLPBMd7Y+CIQomwOgNQ2+rUTylGGzMb0FaJaL4WgPPf5lzkY5KD4V6
-    symbol_server: https://www.myget.org/F/telegram-bot/api/v2/package
+      secure: gSQ82eM3mJYkPfEF2qPLvzEw45dS6C/VAwLHTMn3mRpVMhVybLzBUCWDnP5lAmp6
+    symbol_server: https://www.myget.org/F/telegram-bot-core/api/v2/package
```
Comment (MrRoundRobin) [inline appveyor.yml]: why this Change?

Comment (pouladpld) [inline appveyor.yml]: Welcome Back! There were some issues with builds and you were not around so I tried to setup my own. Feel free to revert it back.
I guess your API token is changed.

Comment (pouladpld) [inline appveyor.yml]: I'll disconnect my AppVeyor from this repo now.

Comment (pouladpld) [inline appveyor.yml]: @MrRoundRobin 
> is this ok to be released before work on API v3.3?

I published 2 versions to `.Core` package so far and think it's better to publish the same versions(see tags and milestones) for original package as well. It wouldn't make users confused about what fix is in what version.
